docs/docsite/rst/dev_guide/developing_modules_general.rst

```diff
@@ -248,33 +248,33 @@ The next step in testing your new module is to consume it with an Ansible playbo
 Testing basics
 ====================
 
-These two examples will get you started with testing your module code. Please review our :ref:`testing <developing_testing>` section for more detailed
+These two examples will get you started with testing your module code. The :ref:`testing <developing_testing>` section has more detailed
 information, including instructions for :ref:`testing module documentation <testing_module_documentation>`, adding :ref:`integration tests <testing_integration>`, and more.
 
 Sanity tests
 ------------
 
-You can run through Ansible's sanity checks in a container:
+You can run through the Ansible sanity checks in a container. By default the test command will run tests on all supported versions of Python. To limit your test run to a specific Python version::
 
-``$ ansible-test sanity -v --docker --python 2.7 MODULE_NAME``
+  $ ansible-test sanity -v --docker default --python 2.7 MODULE_NAME
 
-Note that this example requires Docker to be installed and running. If you'd rather not use a
-container for this, you can choose to use ``--tox`` instead of ``--docker``.
+Note that this example requires Docker to be installed and running. If you prefer, you can use the Python testing wrapper by passing ``--tox`` instead of ``--docker <container_name>``.
 
 Unit tests
 ----------
 
-You can add unit tests for your module in ``./test/units/modules``. You must first setup your testing environment. In this example, we're using Python 3.5.
+You can add unit tests for your module in ``./test/units/modules``. You can run these in a container as well. Be sure to set up your development environment::
```

Comment:
We need to be careful about paths in test instructions. Not all paths are the same for ansible/ansible and collections. For example, unit tests are in `test/units/` for ansible/ansible and in `tests/unit/` for collections.